thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -10,6 +10,7 @@ const ROOTCAUSE_TAB_DIMENSIONS = "dimensions";
 const ROOTCAUSE_TAB_METRICS = "metrics";
 const ROOTCAUSE_TAB_EVENTS = "events";
 
+const ROOTCAUSE_SERVICE_ROUTE = "route";
```

Comment:
Please use single quotes `'` for string in JS. (This is a convention we adopted) 
Applies to all prs.

Reply:
fixed